scripts/load-toxref-vocab.py

```diff
@@ -0,0 +1,158 @@
+import json
+
+# import os
```

Comment:
remove the commented out code, e.g. "# import os", "# os.environ.setdefault", etc. (obv good to keep actual explanatory comments like "# Generate the data fixture...")